docs/content/test_runner/Metrics.md

```diff
@@ -28,4 +28,9 @@ Time from first test execution start to last test execution finished
 
 Total job time (suite time with both delays)
 
+### `device-wasted-relative-median`
```

Comment:
```suggestion
### `device-utilization.median`
```
`.median` - for extensibility in case of adding another aggregates

Reply:
thanks! will do